packages/rest/src/lib/utils/constants.ts

```diff
@@ -24,7 +24,7 @@ export const enum RESTEvents {
 	RateLimited = 'rateLimited',
 }
 
-export const ALLOWED_EXTENSIONS = ['webp', 'png', 'jpg', 'jpeg', 'gif'] as const;
+export const ALLOWED_EXTENSIONS = ['webp', 'png', 'jpg', 'jpeg', 'gif', 'json'] as const;
 export const ALLOWED_SIZES = [16, 32, 64, 128, 256, 512, 1024, 2048, 4096] as const;
 
 export type ImageExtension = typeof ALLOWED_EXTENSIONS[number];
```

Comment:
This shouldn't include .json

Reply:
Lottie files are .json

Reply:
Ok and? Not all image are accepting .json, in fact only the stickers CDN endpoint takes in JSON..

Reply:
I guess an additional type could solve this then 👍

Reply:
An additional type would work for the function parameter at least, however this check would throw an error then:
https://github.com/discordjs/discord.js-modules/blob/c4554d237723fd7ea01c2b1c5c0674d239de5871/packages/rest/src/lib/CDN.ts#L152-L156

Reply:
So do something about it 👀

Reply:
This is the best solution I could come up with, if you know something better lmk